modules/ROOT/pages/index.adoc

```diff
@@ -2,7 +2,12 @@
 ifndef::env-site,env-github[]
```

Comment:
I don't have any specific feedback about the text you use, other than it might need to take a more "primer" approach if our audience does not understand terms like "integration"!
So I did mostly brainstorming work, from the perspective of a first reader and not-Mule-pro.